src/modules/uniplayer/Timeout.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+const Timeout = Composition => class _Timeout extends Component {
```

Comment:
Whoa haven't seen this type of composition. You have an article or something for inspiration?

Reply:
This is where I found this https://stackoverflow.com/a/44691405